test/parallel/test-filehandle-readablestream.js

```diff
@@ -113,60 +113,3 @@ const check = readFileSync(__filename, { encoding: 'utf8' });
 
   await file.close();
 })().then(common.mustCall());
-
-// Make sure that acquiring a ReadableStream 'bytes' stream
-// fails if the FileHandle is already closed.
-(async () => {
-  const file = await open(__filename);
-  await file.close();
-
-  assert.throws(() => file.readableWebStream({ type: 'bytes' }), {
-    code: 'ERR_INVALID_STATE',
-  });
-})().then(common.mustCall());
-
-// Make sure that acquiring a ReadableStream 'bytes' stream
-// fails if the FileHandle is already closing.
-(async () => {
-  const file = await open(__filename);
-  file.close();
-
-  assert.throws(() => file.readableWebStream({ type: 'bytes' }), {
-    code: 'ERR_INVALID_STATE',
-  });
-})().then(common.mustCall());
-
-// Make sure the 'bytes' ReadableStream is closed when the underlying
-// FileHandle is closed.
-(async () => {
-  const file = await open(__filename);
-  const readable = file.readableWebStream({ type: 'bytes' });
-  const reader = readable.getReader({ mode: 'byob' });
-  file.close();
-  await reader.closed;
-})().then(common.mustCall());
-
-// Make sure the 'bytes' ReadableStream is closed when the underlying
-// FileHandle is closed.
-(async () => {
-  const file = await open(__filename);
-  const readable = file.readableWebStream({ type: 'bytes' });
-  file.close();
-  const reader = readable.getReader({ mode: 'byob' });
-  await reader.closed;
-})().then(common.mustCall());
-
-// Make sure that the FileHandle is properly marked "in use"
-// when a 'bytes' ReadableStream has been acquired for it.
-(async () => {
-  const file = await open(__filename);
-  file.readableWebStream({ type: 'bytes' });
-  const mc = new MessageChannel();
-  mc.port1.onmessage = common.mustNotCall();
-  assert.throws(() => mc.port2.postMessage(file, [file]), {
-    code: 25,
-    name: 'DataCloneError',
-  });
-  mc.port1.close();
-  await file.close();
-})().then(common.mustCall());
```

Comment:
Why were all these tests removed, is there a way they can be reworked with the new behavior?

Reply:
When https://github.com/nodejs/node/pull/46933 added a parameter allowing callers to pass the `type` parameter down to the ReadableStream constructor (taking us from always not-`bytes` to optionally `bytes`), some existing test cases were duplicated and slightly modified to test the `bytes` case.

With this PR, we're going to always-`bytes`, so one half of the codepaths in this regard are eliminated, and there are correspondingly redundant tests.